tests/app_test.rb

```diff
@@ -0,0 +1,63 @@
+ENV['RACK_ENV'] = 'test'
+
+require_relative '../app'
+require 'test/unit'
+require 'rack/test'
+require 'pry'
+
+
+configure do
+  DataMapper::setup(:default, "sqlite3:test.db")
+  DataMapper.finalize
+  DataMapper.auto_migrate!
+end
+
+class ApplicationControllerTest < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    @app ||= ApplicationController.new
+  end
+
+  def test_home_page
+    get '/'
+    last_response.body
+    assert last_response.ok?
```

Comment:
Also check some text that you expect in the body? I suppose this just tests the response code to be 200.

Actually a better test for index would be to put some data, then call index and check that the data was stored.